ebusd/CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 <!-- https://developers.home-assistant.io/docs/add-ons/presentation#keeping-a-changelog -->
 
+## TBC
+
+- BREAKING: Remove old style loglevel and logareas option
+- BREAKING: Remove TCP port option.  Port will default to port 8888 internally.  External port can be configured in network options
```

Comment:
User can still edit the external port, this just fixes the internal port number to 8888 so shouldn't have any negative affect